docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
```

Comment:
Maybe a UML-like diagram (a .png picture) showing the relationship between users/roles/services/subservices may help to clarify.